nana/modules/admins.py

```diff
@@ -151,184 +189,146 @@ async def pin_message(client, message):
-            else:
-                await edrep(message, text="`Reply to a user to mute them`")
-                await asyncio.sleep(5)
+            try:
+                if message.reply_to_message:
+                    user_id = message.reply_to_message.from_user.id
+                else:
+                    usr = await client.get_users(message.command[1])
+                    user_id = usr.id
+            except IndexError:
+                await message.reply('some ooga booga')
+                return
+            try:
+                await client.restrict_chat_member(
+                    chat_id=message.chat.id,
+                    user_id=user_id,
+                    permissions=mute_permission
+                )
                 await message.delete()
+            except Exception as e:
+                await edrep(message, text="`Error!`\n"
+                        f"**Log:** `{e}`"
+                    )
+                return
         else:
-            await edrep(message, text="`permission denied`")
-            await asyncio.sleep(5)
-            await message.delete()
+            await edrep(message, text=tld('denied_permission'))
     else:
         await message.delete()
 
 
 @app.on_message(filters.user(AdminSettings) & filters.command("unmute", Command))
 async def unmute(client, message):
-    if message.chat.type in ["group", "supergroup"]:
+    if message.chat.type in ['group', 'supergroup']:
         chat_id = message.chat.id
-        get_group = await client.get_chat(chat_id)
         can_unmute = await admin_check(message)
         if can_unmute:
             try:
                 if message.reply_to_message:
-                    get_mem = await client.get_chat_member(
-                        chat_id, message.reply_to_message.from_user.id
-                    )
-                    await client.restrict_chat_member(
-                        chat_id=message.chat.id,
-                        user_id=message.reply_to_message.from_user.id,
-                        permissions=unmute_permissions,
-                    )
-                    text = "**Unmuted**\n"
-                    text += f"User: [{get_mem.user.first_name}](tg://user?id={get_mem.user.id}) "
-                    text += f"(`{get_mem.user.id}`)\n"
-                    text += f"Chat: `{get_group.title}` (`{chat_id}`)"
-                    await edrep(message, text=text)
-
+                    user_id = message.reply_to_message.from_user.id
                 else:
-                    await edrep(message, text="`Reply to a user to mute them`")
-                    await asyncio.sleep(5)
-                    await message.delete()
+                    usr = await client.get_users(message.command[1])
+                    user_id = usr.id
+            except IndexError:
+                await edrep(message, text='must give a user to unmute')
+                return
+            try:
+                get_mem = await client.get_chat_member(
```

Comment:
Codacy found an issue: [local variable 'get_mem' is assigned to but never used (F841)](https://app.codacy.com/gh/MadeByThePinsHub/Nana-Remix/pullRequest?prid=6330176)